meta-dts-distro/recipes-support/mei-amt-check/mei-amt-check.bb

```diff
@@ -0,0 +1,25 @@
+DESCRIPTION = "AMT status checker for Linux"
+SECTION = "tools"
+HOMEPAGE = "https://github.com/mjg59/mei-amt-check"
+
+LICENSE = "GPL-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=39f5702df600d6d1b1d6d138dc62347d"
+
+SRC_URI = "git://github.com/mjg59/mei-amt-check;protocol=https;branch=master \
+		   file://0001-Makefile-removed-setting-CC-variable.patch \
+		   "
+
+PV = "1.0+git${SRCPV}"
+SRCREV = "ec921d1e0a2ac770e7835589a28b85bc2f15200c"
+
+S = "${WORKDIR}/git"
+
+do_compile () {
+	oe_runmake
```

Comment:
@kotylamichal please use 2 spaces as tab like in other recipes

Reply:
like here https://github.com/Dasharo/meta-dts-ce/pull/2/files#diff-1c81a7aabd49b3985d32e343cf1916180229ee8e270414c5c278ad1095858099R11

Reply:
@TomaszAIR you mean 4 spaces instead of tab?

Reply:
@kotylamichal https://www.openembedded.org/wiki/Styleguide - it may be quite old but worth checking. It mention 4 spaces instead of tab, lot of time I saw also using 2 spaces.

Reply:
@TomaszAIR ok, now should be like in that guide